packages/react-mutation-mapper/src/component/civic/Civic.tsx

```diff
@@ -19,43 +24,41 @@ export function hideArrow(tooltipEl: any) {
     arrowEl.style.display = 'none';
 }
 
-@observer
-export default class Civic extends React.Component<ICivicProps, {}> {
-    @observable tooltipDataLoadComplete: boolean = false;
+export function sortValue(civicEntry: ICivicEntry | null | undefined): number {
+    let score = 0;
 
-    public static sortValue(
-        civicEntry: ICivicEntry | null | undefined
-    ): number {
-        let score: number = 0;
+    if (civicEntry) {
+        score = 1;
+    }
 
-        if (civicEntry) {
-            score = 1;
-        }
+    return score;
+}
 
-        return score;
+export function download(civicEntry: ICivicEntry | null | undefined): string {
+    if (!civicEntry) {
+        return 'NA';
```

Comment:
if theres any parsing of these values, it would be better to use and reuse a constant with value "NA". If it's just for display then NVM.

Reply:
probably no parsing for these values, but not 100% sure. Let's just add a constant in `CivicUtils`, it wouldn't hurt.